src/webapp-lib/app.pug

```diff
@@ -26,8 +26,8 @@ html(lang="en")
       | Initializing ...
 
     //- preflight check comes **after** the startup banner and status info, and before any code that might be syntactically broken.
-    script(type="text/javascript")
-      include:typescript preflight-checks.ts
+    //script(type="text/javascript")
+    //  include:typescript preflight-checks.ts
```

Comment:
@haraldschilly Whenever you have time, can you check why this fails? Just uncomment.

Reply:
I checked this branch out, updated, made sure I work with the updated version, uncommented it:

```
~$ which tsc
/home/user/smc/src/node_modules/.bin/tsc
~$ tsc --version
Version 3.1.6
```

but the app.html page builds fine and the preflight check comment in the js log shows up with the correct version number. what exactly is going wrong? maybe I have something cached somewhere, though.